curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ddbd81294d8ddc1510a8e56.md

```diff
@@ -1,44 +1,14 @@
 ---
 id: 5ddbd81294d8ddc1510a8e56
-title: Step 11
+title: Step 12
 challengeType: 0
-dashedName: step-11
+dashedName: step-12
 ---
 
 # --description--
 
-A link's text must be placed between the opening and closing tags of an anchor (`a`) element.
-
-Here is an example of a link with the text `click here to go to freeCodeCamp.org`:
-
-```html
-<a href="https://www.freecodecamp.org">click here to go to freeCodeCamp.org</a>
-```
-
-Add the anchor text `link to cat pictures` to the anchor element. This will become the link's text.
-
 # --hints--
 
-Your anchor (`a`) element should have an opening tag. Opening tags have this syntax: `<elementName>`.
-
-```js
-assert(document.querySelector('a'));
-```
-
-Your anchor (`a`) element should have a closing tag. Closing tags have a `/` just after the `<` character.
-
-```js
-assert(code.match(/<\/a\>/));
-```
-
-Your anchor (`a`) element's text should be `link to cat pictures`. Make sure to put the link text between the anchor (`a`) element's opening tag and closing tag.
-
-```js
-assert(
-  document.querySelector('a').innerText.toLowerCase().replace(/\s+/g, ' ') ===
-    'link to cat pictures'
-);
-```
```

Comment:
Removing all of the text here is breaking the tests entirely.

Each step needs both a description and at least one hint.

Reply:
Hi.  I've just started using the challenge editor.
Am going through several steps per editing session.
Currently this is still a draft, will mark as ready when all changes are finalised.